x-pack/plugins/apm/public/components/app/Settings/anomaly_detection/update_jobs_callout.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { EuiButton, EuiCallOut } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useState } from 'react';
+import { useApmPluginContext } from '../../../../context/apm_plugin/use_apm_plugin_context';
+import { callApmApi } from '../../../../services/rest/createCallApmApi';
+
+export function UpdateJobsCallout({
+  onUpdateComplete,
+}: {
+  onUpdateComplete: () => void;
+}) {
+  const [loading, setLoading] = useState(false);
+  const { core } = useApmPluginContext();
+
+  function upgradeJobs() {
+    setLoading(true);
+    callApmApi({
+      endpoint: 'POST /internal/apm/settings/anomaly-detection/update_to_v3',
+      signal: null,
+    })
+      .then(() => {
+        core.notifications.toasts.addSuccess({
+          title: i18n.translate(
+            'xpack.apm.jobsList.updateCompletedToastTitle',
+            {
+              defaultMessage: 'Anomaly detection jobs created!',
+            }
+          ),
+          text: i18n.translate('xpack.apm.jobsList.updateCompletedToastText', {
+            defaultMessage:
+              'Your new anomaly detection jobs have been created successfully. You will start to see anomaly detection results in the app within minutes. The old jobs have been closed but the results are still available within Machine Learning.',
+          }),
+        });
+        onUpdateComplete();
+      })
+      .catch(() => {
+        setLoading(false);
+      });
```

Comment:
Shouldn't we show a warning toast if the job creation fails? And shouldn't `setLoading(false);` be called regardless if it failed or not?

```suggestion
      .catch(() => {
        core.notification.toasts.addWarning({...})
      })
      .finally(() => {
        setLoading(false);
      })
```